README.md

```diff
@@ -264,24 +265,31 @@ The following steps describe the minimal steps required to successfully log in t
 1. In the `uid2_selfserve` database, observe that `dbo.users` now contains a row with with the details you just filled out.
 1. Approve your account by updating the `status` of the row in `dbo.participants` that corresponds to your new user, i.e.
 
-    ```
-    use [uid2_selfserve]
+   ```
+   use [uid2_selfserve]
 
-    declare @email as nvarchar(256) = '<Enter your email here>'
+   declare @email as nvarchar(256) = '<Enter your email here>'
 
-    update p
-    set status = 'approved'
-    from dbo.participants p
-    left join dbo.users u
-      on p.id = u.participantId
-    where u.email = @email
-    ```
+   update p
+   set status = 'approved'
+   from dbo.participants p
+   left join dbo.users u
+     on p.id = u.participantId
+   where u.email = @email
+   ```
 
 1. Assign yourself the `api-participant-member` role by following these steps: [Assign Role to a Particular User](./KeycloakAdvancedSetup.md#assign-role-to-a-particular-user)
 1. Run the Admin service locally by following [Connecting to local Admin service](#connecting-to-local-admin-service)
 1. Optionally give your user access to the [Admin screens/routes](#admin-screensroutes)
 1. Return to the UI and you should be good to go!
 
+#### \* Notes for Mac OSX Development:
```

Comment:
Was the `\*` intentional here? I think using `####` is sufficient 
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/ca02c069-e604-4d09-8dfd-658df11cba9d)

Reply:
removed